metricbeat/module/kubernetes/state_deployment/data.go

```diff
@@ -21,11 +21,14 @@ func eventMapping(families []*dto.MetricFamily) ([]common.MapStr, error) {
 			if deployment == "" {
 				continue
 			}
-			event, ok := eventsMap[deployment]
+			namespace := util.GetLabel(metric, "namespace")
+			deploymentKey := namespace + "::" + deployment
```

Comment:
@exekias Does this mean we have `::` in the field key?

Reply:
No, this is a key in the `eventMap`, we use it to correlate all different metrics into a single event, then we ship events, forgetting about the keys in the map

Reply:
got it, thanks.